infra/feast-operator/config/samples/v1alpha1_featurestore_pvc_persistence.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: sample-pvc-persistence
+spec:
+  feastProject: my_project
+  services:
+    onlineStore:
+      persistence:
+        file:
+          path: online_store.db
+          pvc:
+            ref:
+              name: online-pvc
+            mountPath: /data/online
+    offlineStore:
+      persistence:
+        file:
+          type: duckdb
+          pvc:
+            create:
+              storageClassName: standard
+              resources:
+                requests:
+                  storage: 5Gi
+            mountPath: /data/offline
+    registry:
+      local:
+        persistence:
+          file:
+            path: registry.db
+            pvc:
+              create: {}
+              mountPath: /data/registry
```

Comment:
why `create` section empty here, is there related docs added for this configuration. and looks the settings not consistent with each type

For example registry 
```        persistence:
          file:
            path: registry.db
            pvc:
              create: {}
              mountPath: /data/registry
```
online
```        file:
          path: online_store.db
          pvc:
            ref:
              name: online-pvc
            mountPath: /data/online

```

Reply:
i think he's demonstrating 3 different types of pvc configurations. all 3 feast service types can be configured similarly. an empty `create:` tells the operator that the user wants a pvc, but doesn't want to define it. the operator will apply defaults during the creation.

Reply:
There is no doc apart from the CRD definition, which has some comments on it.
wrt your questions:
- pvc can have either create or ref, and create has default fields in case they are omitted
Anyway, to avoid misunderstandings, I will add the other settings too and avoid defaults

Reply:
imo, CRD should be the doc of choice for API. @dmartinol i like the file in its current state, shows the user all their options.

Reply:
agree, reverted previous change